src/main/java/net/sf/jabref/logic/msbib/MSBibConverter.java

```diff
@@ -135,10 +127,6 @@ public static MSBibEntry convert(BibEntry entry) {
     }
 
     private static String removeLaTeX(String text) {
-        // TODO: just use latex free version everywhere in the future
-        String result = new RemoveBrackets().format(text);
```

Comment:
Why did you remove the bracketsformatter here? I added it so we dont end up wit `{ACM} Cooperation` in Word bibliographies. As long as we don't have a LaTeX-free BibEntry version we should keep it that way?